project/Dependencies.scala

```diff
@@ -54,6 +51,17 @@ object Dependencies {
 
   // format: on
 
+  val nettyTcnativeAssemblyExclusions: Seq[String] =
+    if(osClassifier == "osx-x86_64")
+      Seq("netty-tcnative-openssl-static-2.0.9.Final-SNAPSHOT-linux-x86_64.jar")
```

Comment:
I don't understand why this is `-linux` if the `osClassifier` is `osx-x86_64`.

Reply:
Because it's an artefact **exclusion** list. We want to exclude the Linux jar on a macOS.